examples/plot_investigating_dirty_categories.py

```diff
@@ -122,24 +123,50 @@
 f2.tight_layout()
 
 ########################################################################
-# As shown in the previous plot, we see that "communication Equipment technician"'s
-# nearest neighbor is "telecommunication technician", although it is also
+# As shown in the previous plot, we see that the nearset neighbor of "Communication Equipment Technician"
+# is "telecommunication technician", although it is also
 # very close to senior "supply technician": therefore, we grasp the
 # "communication" part (not initially present in the category as a unique word)
 # as well as the technician part of this category.
 
 
-
 #########################################################################
 # Encoding categorical data using SimilarityEncoder
 # -------------------------------------------------
 #
 # A typical data-science workflow uses one-hot encoding to represent
 # categories.
+from sklearn.preprocessing import CategoricalEncoder
+
+n_obs, n_categories = 20, 20
+# employee_position_titles = values[
+#     'Employee Position Title'].sort_values().to_frame()
+# or
+# employee_position_titles=values[
+#     'Employee Position Title'].sample(n=n_observations)?
+categorical_encoder = CategoricalEncoder(encoding='onehot-dense')
+one_hot_encoded = categorical_encoder.fit_transform(
+    sorted_values[:n_obs].reshape(-1, 1))
+f3, ax3 = plt.subplots(figsize=(6, 6))
+cax3 = ax3.matshow(one_hot_encoded[:n_obs, :n_categories])
+f3.colorbar(cax3)
+f3.suptitle('Employee Position Title values, one-hot encoded')
+ax3.xaxis.tick_bottom()
+f3.tight_layout()
 
 #########################################################################
 # The corresponding is very sparse
 #
 # SimilarityEncoder can be used to replace one-hot encoding capturing the
 # similarities:
 
+f4, ax4 = plt.subplots(figsize=(6, 6))
+cax4 = ax4.matshow(transformed_values[:n_obs, :n_obs])
+f4.colorbar(cax4)
+f4.suptitle('Employee Position Title values, similarity encoded')
+ax4.xaxis.tick_bottom()
+f4.tight_layout()
+
+#########################################################################
+# you can check out how similarity encoding impacts prediction performance
```

Comment:
I would rephrase as following:

"Other examples in the dirty_cat documentation show how similarity encoding impacts prediction performance."